NimCheck.py

```diff
@@ -96,7 +97,7 @@ def run(self, show_error_list=True):
         # Run 'nim check' on the current view and retrieve the output.
         output, returncode = yield Thread(
             target=run_nimcheck,
-            args=(view.file_name(), this.send)
+            args=(view.file_name(), this.send, this.verbosity)
```

Comment:
It should be "self.verbosity", I think. "this" refers to the generator instance of the command.